tools/killswitch/src/output.rs

```diff
@@ -0,0 +1,287 @@
+use crate::{errors::Error, killswitch::Channel};
+use ethers::prelude::H256;
+use nomad_core::TxOutcome;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+/// KillSwitch response showing success / failure of configuration
+/// and tx submission. Gets serialized to json
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct Output {
+    /// The original command `killswitch` was run with
+    pub command: String,
+    /// The success / failure message
+    pub message: Message,
+}
+
+/// A wrapper for success / failure messages
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub(crate) enum Message {
+    /// An wrapper for a single error we bailed on
+    SimpleError(String),
+    /// A full results message
+    FullMessage(Homes),
+}
+
+impl From<Error> for Message {
+    /// Convert a blocking `Error` to `Message`
+    fn from(error: Error) -> Self {
+        Message::SimpleError(format!("{}", error))
+    }
+}
+
+/// Map of homes by name
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct Homes {
```

Comment:
Nit: can we rename `ConnectedHomes` to avoid confusion with actual `Homes` object

Reply:
That's to match the json response structure `{"homes": {...`. The alternative to a wrapper struct is mixing in ad-hoc `json!` -- `json!({"homes": {...}})` -- which is undesirable. Ditto for the other cases.